llvm/lib/CodeGen/GlobalISel/LegalityPredicates.cpp

```diff
@@ -204,6 +204,8 @@ LegalityPredicate LegalityPredicates::memSizeInBytesNotPow2(unsigned MMOIdx) {
 LegalityPredicate LegalityPredicates::memSizeNotByteSizePow2(unsigned MMOIdx) {
   return [=](const LegalityQuery &Query) {
     const LLT MemTy = Query.MMODescrs[MMOIdx].MemoryTy;
+    if (Query.MMODescrs[MMOIdx].MemoryTy.isScalableVector())
+      return true;
```

Comment:
Is this needed for addsat?

Reply:
Nope. Probably left over from debugging.